### ISSUE TITLE
disposal pipes are not insulated from cold/pressure above surface

### ISSUE THREAD

**Issue by Spacedong** [HippieStationCode/HippieStation13#2704]
despite being below the station's floor tiles and basically made out of fucking steel you'll still burn to death if you ride them underneath a fire, no matter how big it is

**Comment by ArcLumin** [comment]
This is especially apparent on the mining asteroid, the disposals linking areas are in an unpressurized, cold environment, so you'll take damage.

Always saw this as a feature, though.

**Comment by Spacedong** [comment]
kinda doesn't make sense for on-station though considering they're buried underneath the metal floors and shouldn't really be affected by things going on above ground

**Comment by Spacedong** [comment]
i dont think we can make them have their own atmos like atmospheric pipes tho

**Comment by GeneralGoose** [comment]
i thought they heat with fire and you get roasted alive while inside

**Comment by Spacedong** [comment]
no thats not a feature thats a bug/oversight/intended but its still dumb they should be atmos proof while under tiles and not when exposed
